.ko.yaml

```diff
@@ -2,4 +2,4 @@ defaultBaseImage: gcr.io/distroless/static:nonroot
 baseImageOverrides:
   # git-init uses a base image that includes Git, and supports running either
   # as root or as user nonroot with UID 65532.
-  github.com/tektoncd/pipeline/cmd/git-init: gcr.io/tekton-nightly/github.com/tektoncd/pipeline/git-init-build-base:latest
+  github.com/tektoncd/pipeline/cmd/git-init: ghcr.io/mattmoor/git # DO NOT SUBMIT ghcr.io/distroless/git
```

Comment:
This must be reverted before things go in, which requires an `apko` patch release and a kick to the `git` build.

Reply:
Throwing a `[WIP]` on this until this is reverted

Reply:
Ok, this is done, and the new images are published with `/root` :party-matt: